### PR TITLE
testeth refactoring

### PR DESCRIPTION
**Pull request by winsvega**
split large TestHelper.cpp into separate files.

there was several class realizations in one single TestHelper.cpp making it about 1000+ lines long. 
So the point of this PR is to separate those realization. like ImportTest, TestOutputHelper, Options  into different files and headers.

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -9,11 +9,12 @@ macro (add_sources)
 	endforeach()
 endmacro()
 
+add_subdirectory(libtesteth)
 add_subdirectory(fuzzTesting)
 add_subdirectory(libethcore)
 add_subdirectory(libethereum)
-if (NOT EMSCRIPTEN)		
-    add_subdirectory(libtestutils)		
+if (NOT EMSCRIPTEN)
```
Comment (chfast) [inline test/CMakeLists.txt]: You can remove EMSCRIPTEN check. It was for solidity.

File: test/fuzzTesting/fuzzHelper.cpp [hunk 2]
```diff
@@ -19,12 +19,12 @@
...
  */
 
-#include "fuzzHelper.h"
-
 #include <chrono>
 #include <boost/random.hpp>
 #include <boost/filesystem/path.hpp>
 #include <libevmcore/Instruction.h>
+#include <test/libtesteth/TestOutputHelper.h>
+#include <test/fuzzTesting/fuzzHelper.h>
```
Comment (chfast) [inline test/fuzzTesting/fuzzHelper.cpp]: There is a rule to include matching header (name.cpp -> name.hpp) as the first include (as it was here). I think we should keep it.

File: test/libtesteth/TestOutputHelper.h [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file TestOutputHelper.h
```
Comment (chfast) [inline test/libtesteth/TestOutputHelper.h]: Don't repeat file name after `@file`!

Comment (winsvega) [inline test/libtesteth/TestOutputHelper.h]: no filenames after @file  ?

Comment (chfast) [inline test/libtesteth/TestOutputHelper.h]: It is optional and should be only needed when documenting **other** file. https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdfile

**Pull request merged**